android/app/src/main/java/app/lunes/MainApplication.java

```diff
@@ -4,6 +4,7 @@
 import android.content.Context;
```

Comment:
renamed this folder, as the path for saving files to app specific storage on android will equal this.